### PR TITLE
e2e for ExcelPlugin as ITN

### PR DESCRIPTION
**Pull request by AnkitCLI**
E2E test covered as ITN for excel input reader.

### CODE REVIEW

**Review by itsankit-google — COMMENTED** [review]
Can you please add it as required tests for PR review?

I do not see the test running in cucumber report: https://storage.googleapis.com/e2e-tests-cucumber-reports/hydrator-plugins/refs/pull/1855/merge/plugin/core-plugins/target/cucumber-reports/advanced-reports/cucumber-html-reports/overview-features.html

**Comment by AnkitCLI** [comment]
Actually they are running u can see it here =https://storage.googleapis.com/e2e-tests-cucumber-reports/hydrator-plugins/refs/pull/1855/merge/plugin/core-plugins/target/cucumber-reports/advanced-reports/cucumber-html-reports/report-feature_12_2189426909.html.

**Comment by itsankit-google** [comment]
> Actually they are running u can see it here =https://storage.googleapis.com/e2e-tests-cucumber-reports/hydrator-plugins/refs/pull/1855/merge/plugin/core-plugins/target/cucumber-reports/advanced-reports/cucumber-html-reports/report-feature_12_2189426909.html.

Cool, thanks for the pointer.

File: core-plugins/src/e2e-test/resources/pluginParameters.properties [hunk 1]
```diff
@@ -291,3 +291,12 @@ quotedValueDelimitedTestFiles=dummy
 fileSinkTargetBucket=file-plugin-output
 recursiveTest=dummy
 testOnCdap=true
+
+## EXCEL-PLUGIN-PROPERTIES-START ##
+excelTestFile=dummy
+excelFile=testdata/exceldata/testExcel.xlsx
+Pattern=.*\.xlsx$
```
Comment (itsankit-google) [inline core-plugins/src/e2e-test/resources/pluginParameters.properties:298]: Does the plugin only support `xlsx` format? 

Or will the tests for other formats come later?

Comment (AnkitCLI) [reply core-plugins/src/e2e-test/resources/pluginParameters.properties:298]: yes this only support xls format. And as per ITN's we have to add for this one only.

Comment (itsankit-google) [reply core-plugins/src/e2e-test/resources/pluginParameters.properties:298]: > yes this only support xls format. And as per ITN's we have to add for this one only.

`xls` and `xlsx` are different formats? Which one are we looking to add the test for?

Comment (AnkitCLI) [reply core-plugins/src/e2e-test/resources/pluginParameters.properties:298]: oh sorry we checked for xlsx. The simple use case we are checking is to send data from excel plugin to sink.So we just took the example for xlsx which is newer one (there is nothing mentioned in repo to use which format).

**Review by itsankit-google — APPROVED** [review]

**Pull request merged**